api/database/factories/PoolFactory.php

```diff
@@ -28,25 +28,27 @@ class PoolFactory extends Factory
      */
     public function definition()
     {
+        $userId = User::inRandomOrder()
```

Comment:
Yeah, this will make the seeded data a little more realistic.
[pool owner is admin](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6736/commits/16d7166ee9dac2459f4a7949897ef2a3db7104ea)